boards/arm/stm32f746g_disco/stm32f746g_disco.dts

```diff
@@ -18,6 +18,10 @@
 		zephyr,flash = &flash0;
 		zephyr,dtcm = &dtcm;
 	};
+	
+	aliases {
+		kscan0 = &touch_controller;
```

Comment:
Please move this to the existing aliases section below.

Reply:
Oops...missed that as it was lower down.